pagy.gemspec

```diff
@@ -1,25 +1,18 @@
-
 lib = File.expand_path('../lib', __FILE__)
 $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
-require 'pagy'
-require 'date'
+require 'pagy/version'
```

Comment:
loading all of pagy is not necessary ... also allows only version to be loaded

Reply:
While true, the "all of pagy" is actually so small that would not bother anyone/anything

Reply:
just a pattern I use/saw a lot and also makes https://github.com/ddnexus/pagy/pull/29 smaller :)